lib/msal-core/src/telemetry/TelemetryTypes.ts

```diff
@@ -9,14 +9,20 @@ export type TelemetryEmitter = (events: Array<object>) => void;
 
 // SDK SHOULD BE DEFAULTED and Pulled from Package
 export interface TelemetryPlatform {
-    sdk: string;
-    sdkVersion: string;
+    sdk?: string;
+    sdkVersion?: string;
     applicationName: string;
     applicationVersion: string;
+    networkInformation?: NetworkInformation
```

Comment:
Does JS work with mandated params in between and optionals spread out? In function signatures they need to come after required parameters, I am not sure about object signatures.

Reply:
not clear to me what is being asked here :)

Reply:
I think @sameerag makes a good point. Usually optional parameters come after the required ones. Not sure if its strictly required for interfaces like it is for functions but it's probably good form to do so.

Reply:
Yeah that matters for function signatures, but doesn't matter here, as these values are key-based, not positioned based.